testsuite/module-publish/src/packages/experimental_usecases/sources/existence.move

```diff
@@ -0,0 +1,103 @@
+module 0xABCD::existence {
+    use std::signer;
+    use std::vector;
+    use std::string::{String, utf8};
+    use aptos_std::table;
+    use aptos_std::table::{Table};
+    use aptos_framework::event;
+
+    const ENOT_AUTHORIZED: u64 = 1;
+
+    struct A has key, store, copy, drop {
+        a1: u64,
+        a2: String,
+    }
+
+    struct B has key, store, copy, drop {
+        va: vector<A>,
+        name: String,
+        super_a: A,
+    }
+
+    struct C has key {
+        b1: B,
+        bt: Table<B, String>,
```

Comment:
Btw, in terms of layout this is just a u128 handle, so if the goal is get resource size bigger, large table will not work (every item is its own lot in storage, it is not owned by this resource). But if you use some smart data structure, `SimpleMap` or something like this, it will be large. Not sure what your goal here was